pkg/devfile/detect.go

```diff
@@ -140,33 +140,37 @@ func AnalyzePath(a Alizer, localpath, context, devfileRegistryURL string, devfil
 		if dockerfileUri, err := SearchForDockerfile(devfile); err != nil {
 			return err
 		} else if len(dockerfileUri) > 0 {
+			dockerfileContextMapFromRepo[context] = path.Join(context, dockerfileUri)
```

Comment:
Dumb question: Can `dockerfileUri` be a URL in any case? E.g. some scenario where we wouldn't want to automatically append the context?

Reply:
you are right. in our api reference: `URI Reference of a Dockerfile. It can be a full URL or a relative URI from the current devfile as the base URI.`

Reply:
true, my bad